File: backend/models/statisticsModel.js
const mongoose = require('mongoose');
const ModelFactory = require('./modelFactory');

const StatisticsSchema = new mongoose.Schema({
  statistics_id: String,
  child_death: Number,
  soldier_death: Number,
  adult_death: Number,
  injured: Number,
  statistics_country_id: String,
  statistics_war_id: String,
});

const Statistics = ModelFactory.createModel('statistics', StatisticsSchema);

module.exports = Statistics;

File: InfoWarOSMobilUygulamasi/InfoWar/InfoWar/Model/StatisticsModel.swift
//
//  StatisticsModel.swift
//  InfoWar
//
//  Created by Serhat  Şimşek  on 4.01.2024.
//

import Foundation

struct StatisticsModel: Decodable {
    let id, statisticsId: String?
    let childDeath, soldierDeath, adultDeath, injured: Int?
    let statisticsCountryId, statisticsWarId: String?
}
